crates/biome_rowan/src/syntax/node.rs

```diff
@@ -1206,3 +1206,29 @@ impl<L: Language> SyntaxNodeWithOffset<L> {
         }
     }
 }
+
+/// Marker trait to prevent unrelated types to be contained in the [`crate::ErasedSyntaxNode`] struct.
+pub trait AsSyntaxNode: Any {}
+
+impl<L: Language + 'static> AsSyntaxNode for SyntaxNode<L> {}
+
+/// Opaque struct for [`SyntaxNode`] without the `L: Language` constraint.
+#[derive(Debug)]
+pub struct AnySyntaxNode {
```

Comment:
What is the advantage of using `AnySyntaxNode` over the `SendNode` type we already have? In the Grit plugin we can say that the nodes are directly converted to a `SendNode` anyway, so why don't we just use `SendNode` everywhere instead of introducing a new type here?

Reply:
Yes I introduced `AnySyntaxNode` because it's safe to use with any descendants but `!Send`. I am not sure whether the limitation of `SendNode` can be lifted and merged into `AnySyntaxNode`.